turbo.json

```diff
@@ -2,9 +2,11 @@
   "$schema": "https://turbo.build/schema.json",
   "globalEnv": ["CLOUDFLARE_ACCOUNT_ID", "CLOUDFLARE_API_TOKEN"],
   "tasks": {
+    "build": {},
```

Comment:
I'd recommend adding some outputs here so that caching works correctly during local development:
```json
"build": {
	"outputs": ["dist/**", ".astro/**", ".wrangler/deploy/config.json"], // any others you can think of
},
```